localization/autoware_ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -721,10 +721,10 @@ void NDTScanMatcher::transform_sensor_measurement(
   }
 
   const geometry_msgs::msg::PoseStamped target_to_source_pose_stamped =
-    autoware_utils::transform2pose(transform);
+    autoware_utils_pcl::transform2pose(transform);
```

Comment:
```suggestion
    autoware_utils_geometry::transform2pose(transform);
```
Is it autoware_utils_geometry?

Reply:
That's right.
https://github.com/autowarefoundation/autoware_utils/blob/main/autoware_utils_geometry/src/geometry/geometry.cpp#L126